.env_sample

```diff
@@ -0,0 +1,5 @@
+export GITHUB_CLIENT_ID=d6fca75c63daa014c187
```

Comment:
are these valid? 💭 
or did you ... you recycled the "invalidated" ones https://github.com/search?q=org%3Adwyl+d6fca75c63daa014c187&type=Code 👍

Reply:
I've recycled them, they are not valid